sources/packages/web/src/views/student/AppStudent.vue

```diff
@@ -1,76 +1,89 @@
 <template>
   <IdleTimeChecker :clientIdType="ClientIdType.Student">
     <!-- Adding overflow:visible to allow the use of the Prime Vue
-  floating menu while Veutify component is not ready.  -->
-    <v-app-bar dense flat app style="overflow: visible">
-      <BCLogo subtitle="Student Application" @click="logoClick"></BCLogo>
-      <v-spacer></v-spacer
-      ><v-btn
-        v-if="hasAuthenticatedStudentAccount"
-        text
-        @click="
-          $router.push({ name: StudentRoutesConst.STUDENT_APPLICATION_SUMMARY })
-        "
-        >Applications</v-btn
+  floating menu while Vuetify component is not ready.  -->
+    <v-app-bar color="white">
+      <b-c-logo subtitle="Student Application" @click="logoClick" />
+      <v-spacer />
+      <v-btn-toggle
+        selected-class="active-btn label-bold"
+        v-model="toggleNav"
+        class="navigation-btn float-left"
       >
-      <v-btn
-        v-if="hasAuthenticatedStudentAccount"
-        text
-        @click="$router.push({ name: StudentRoutesConst.NOTIFICATIONS })"
-        prepend-icon="fa:far fa-bell"
-      >
-        Notifications</v-btn
-      >
-      <v-btn
-        v-if="hasAuthenticatedStudentAccount"
-        text
-        @click="
-          $router.push({ name: StudentRoutesConst.STUDENT_FILE_UPLOADER })
-        "
-        prepend-icon="fa:far fa-file-alt"
-        >File Uploader</v-btn
-      >
-      <v-btn
-        v-if="hasAuthenticatedStudentAccount"
-        text
-        @click="
-          $router.push({ name: StudentRoutesConst.STUDENT_REQUEST_CHANGE })
-        "
-        prepend-icon="fa:far fa-hand-paper"
-        >Request a Change</v-btn
-      >
-      <v-btn
-        v-if="hasAuthenticatedStudentAccount"
-        text
-        @click="$router.push({ name: StudentRoutesConst.STUDENT_PROFILE_EDIT })"
-        >Profile</v-btn
-      >
-      <v-menu v-if="isAuthenticated">
-        <template v-slot:activator="{ props }">
-          <v-btn
-            class="mr-5"
-            icon="fa:fa fa-user"
-            variant="outlined"
-            elevation="1"
-            color="secondary"
-            v-bind="props"
-          ></v-btn>
-        </template>
-        <v-list>
-          <template v-for="(item, index) in menuItems" :key="index">
-            <v-list-item :value="index">
-              <v-list-item-title @click="item.command">
-                <span class="label-bold">{{ item.label }}</span>
-              </v-list-item-title>
-            </v-list-item>
-            <v-divider
-              v-if="index < menuItems.length - 1"
-              :key="index"
-              inset
-            ></v-divider>
+        <v-btn
+          v-if="hasAuthenticatedStudentAccount"
+          class="nav-item-label"
+          variant="text"
+          :to="{
+            name: StudentRoutesConst.STUDENT_DASHBOARD,
+          }"
+          prepend-icon="mdi-home-outline"
```

Comment:
My understanding of the agreement for icons libraries is:
1. We should use FontAwesome as the first option;
2. If FontAwesome does not have the proper icon we should use a fallback;
3. We should not mix FontAwesome and Material design in places like menus because their size does not match and they look different.
@hellolynn-tbtb FYI

Reply:
![image](https://user-images.githubusercontent.com/77353155/195957225-abba3155-33ba-4c87-a091-2ed22723e178.png)

Reply:
As we talked, please check with @hellolynn-tbtb if the UI looks ok with the icons as they are 😉

Reply:
checked with @hellolynn-tbtb. we have only sold the home icon in front awesome. so I extended the btn and added size "30" for the icon as @hellolynn-tbtb recommended.